docs/backend/api-definitions-guidelines.md

```diff
@@ -0,0 +1,58 @@
+# API Definitions Guidelines
+
+This document describes how service APIs will be managed for `InstructLab` and the sub-components of the `InstructLab` backend.
+
+## What parts of InstructLab need service APIs?
+
+There are two primary classes of service APIs needed to support InstructLab:
+
+* `Platform APIs`: These are APIs that are ignorant of `InstructLab` and provide generic AI platform capabilities (e.g., Fine Tuning, SDG, Eval)
+* `InstructLab APIs`: These are the APIs that reflect the user-facing functionality of `InstructLab` itself. They are aware of the end-to-end `InstructLab` workflow.
+
+The `InstructLab APIs` are essential for hosting `InstructLab` as a service in a repeatable way. The `Platform APIs` are critical for component reuse and extensibility (e.g., new SDG algorithms for new taxonomy data types), but they are not strictly required for hosting `InstructLab` as a service.
+
+## How will service APIs be defined?
+
+Service APIs will be defined using [OpenAPI](https://www.openapis.org/) format in [YAML](https://yaml.org/). For structural and style guidelines, see [api-definitions](../../api-definitions/README.md).
+
+## Where will service API definitions live?
+
+Service API definitions will live a new repository github.com/instructlab/service-api-definitions. This repo will have two primary responsibilities:
```

Comment:
I would prefer specification over definitions

Reply:
@hickeyma, can you elaborate on that preference in light of the definition vs specification article? My thought is that this new repo would _explicitly_ house `definitions` (for machine consumption). Are you thinking that it would also house `specifications` (for human consumption)?

Reply:
Ah, good point. I think you're right that the `kubernetes` community uses `spec` to mean what OpenAPI calls `Definition`, though I suppose it's also a little muddy with `CustomResourceDefinition` which is the thing that defines the `schema` for a "thing" and then `spec` being the canonical name for the field within that `schema` where a user would create an instance of that "thing." I that vein, it still feels like `Definition` is the consistent word where the `schema` is defined.

That said, I'm a big fan of putting a `Cares About` number (1-10) on this kind of opinion and then just taking the highest one.

**CA: 2**

Reply:
Looks like we went with `instructlab/openai`. I'm fine with that.